applications/GeoMechanicsApplication/tests/test_strain_measures.py

```diff
@@ -0,0 +1,120 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsStrainMeasureTests(KratosUnittest.TestCase):
+    """
+    This class contains tests which check the displacement result of a column with linear elastic and linear or Hencky ( = natural = logarithmic ) strains
```

Comment:
Nice work; just a small suggestion. In my opinion, at the beginning of the test, the analytical solution can be better described as follows:

```suggestion
    This class contains tests which check the displacement result of a column with linear elastic and linear or Hencky ( = natural = logarithmic ) strains
The analytical solution for these tests is:

For linear strain: ∆H = H0 * (σ / E)
For logarithmic strain: ∆H = H0 * (exp (σ / E) -1)

∆H : top total displacement (m)
H_0 : Original height of column (m)
σ : Compressive stress (N/m2)
E : Young's Modulus (N/m2)'
```

Reply:
Very good suggestion, I have committed it.